configure

```diff
@@ -1699,6 +1705,14 @@ Some influential environment variables:
               capable of building the package. Compiler and feature detection
               tests will be invalid. All options must be configured manually.
               One of: yes,no. (Default: no)
+  OMR_PRODUCT_DESCRIPTION
```

Comment:
This does not appear to match the text is configure.ac.  Can you amend the commit to check in a version of configure that matches configure.ac.  Thanks.